lib/schedule_events.js

```diff
@@ -71,7 +71,8 @@ class ScheduleEvents {
       Rule: params.ScheduleName,
       Targets: [{
         Arn: params.FunctionArn,
-        Id: this._functionName(params)
+        Id: this._functionName(params),
+        Input: JSON.stringify(params.Input)
```

Comment:
We may want to have this backwards compatible by making this optional, i.e. `params.hasOwnProperty('Input') ?
 JSON.stringify(params.Input) : ""`